vectors/nexo/information.json

```diff
@@ -0,0 +1,3 @@
+{
+	"https://nexo.io"
```

Comment:
This is invalid JSON. It misses the domain key. Furthermore, https:// can be removed.